include/OutboundEmail/OutboundEmail.php

```diff
@@ -199,40 +199,56 @@ function getUserMailers($user) {
         		      $autoCreateUserSystemOverride = TRUE;
         		 }
 
-                //Substitute in the users system override if its available.
-                if($userSystemOverride != null)
-        		   $system = $userSystemOverride;
-        		else if ($autoCreateUserSystemOverride)
-        	       $system = $this->createUserSystemOverrideAccount($user->id,"","");
-
-			    $isEditable = ($system->type == 'system') ? FALSE : TRUE; //User overrides can be edited.
-
-                if( !empty($system->mail_smtpserver) )
-				    $ret[] = array('id' =>$system->id, 'name' => "$system->name", 'mail_smtpserver' => $system->mail_smtpdisplay,
-								   'is_editable' => $isEditable, 'type' => $system->type, 'errors' => $systemErrors);
+                // Substitute in the users system override if its available.
+                if($userSystemOverride != null) {
+                    $system = $userSystemOverride;
+                }
+        		else if ($autoCreateUserSystemOverride) {
+                    $system = $this->createUserSystemOverrideAccount($user->id, "", "");
+                }
+                // User overrides can be edited.
+                $isEditable = ($system->type == 'system' || $system->type == 'system-override') ? FALSE : TRUE;
```

Comment:
`$isEditable = ($system->type == 'system' || $system->type == 'system-override') ? FALSE : TRUE;`
is the same as simply
`$isEditable = !($system->type == 'system' || $system->type == 'system-override')`
which is the same as 
`$isEditable = ($system->type != 'system' && $system->type != 'system-override')`
... I would use this last one, but use whichever you prefer :-)